recipes/smefit/recipe.yaml

```diff
@@ -0,0 +1,66 @@
+schema_version: 1
+
+context:
+  name: smefit
+  version: "3.0.1"
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/smefit-${{ version }}.tar.gz
+  sha256: e0dae6da2dce39eabe1775332e7a0357d4e561921380209a16fbd01a4d49477f
+
+build:
+  number: 1
```

Comment:
```suggestion
  number: 0
```

Reply:
Ah sorry! That was from me copy-pasting as I don't think Grayskull knows about v1 format yet.